subsys/bluetooth/audio/Kconfig.has

```diff
@@ -70,24 +27,6 @@ config BT_HAS_PRESET_SUPPORT
 
 if BT_HAS_PRESET_SUPPORT
 
-if BT_HAS_HEARING_AID_BINAURAL
-
-config BT_HAS_IDENTICAL_PRESET_RECORDS
-	bool "Identical preset records in Binaural Hearing Aid Set"
-	help
-	  Set if the list of preset records is identical to the list of preset records
-	  on the other member in the Binaural Hearing Aid Set.
-	  This option sets Independent Presets field in Hearing Aid Features to 0b0.
-
-config BT_HAS_PRESET_SYNC_SUPPORT
-	bool "Preset synchronization support"
-	depends on BT_HAS_IDENTICAL_PRESET_RECORDS
-	help
-	  Set if the hearing aid has support for relaying active preset changes to the other
-	  member in the Binaural Hearing Aid Set.
-
-endif # BT_HAS_HEARING_AID_BINAURAL
-
```

Comment:
> Since the setting of these values are now moved to the register par, should these be moved to, and used in, the sample Kconfig and implementation?

It could make sense, yes.  I would then rather do it together with adding something around it for the BSIM test in a subsequent commit.